Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm

```diff
@@ -3054,6 +3054,20 @@ static void formatForDebugger(const VisiblePositionRange& range, char* buffer, u
 
 - (AXTextMarkerRangeRef)textMarkerRangeAtTextMarker:(AXTextMarkerRef)textMarker forUnit:(TextUnit)textUnit
 {
+#if ENABLE(AX_THREAD_TEXT_APIS)
+    if (AXObjectCache::useAXThreadTextApis()) {
+        AXTextMarker inputMarker { textMarker };
+        switch (textUnit) {
+        case TextUnit::LeftWord:
+            return inputMarker.wordRange(WordRangeType::Left).platformData().autorelease();
+        case TextUnit::RightWord:
+            return inputMarker.wordRange(WordRangeType::Right).platformData().autorelease();
+        default:
+            // TODO: Not Implemented!
+            break;
+        }
+    }
+#endif
```

Comment:
For longer compiler directives like this, it's nice to prefix `#endif` with the matching `#if` condition, e.g.:

`#endif // ENABLE(AX_THREAD_TEXT_APIS)`